app/grants/clr.py

```diff
@@ -243,14 +243,22 @@ def calculate_clr_for_donation(donation_grant, donation_amount, total_pot, base_
     print('error: could not find grant in final grants_clr data')
     return (None, None)
 
-def predict_clr(random_data=False, save_to_db=False, from_date=None):
+def predict_clr(random_data=False, save_to_db=False, from_date=None, clr_type=None, network='mainnet'):
     # setup
     clr_calc_start_time = timezone.now()
 
     # get all the eligible contributions and calculate total
     contributions = Contribution.objects.prefetch_related('subscription').filter(created_on__gte=CLR_START_DATE, created_on__lte=from_date)
     debug_output = []
-    grants = Grant.objects.all()
+
+    grants = Grant.objects.none()
```

Comment:
is this line some kind of fallback? It seems the following if else block would assign grants to something without needing this line

Reply:
Yeah ! Just kept it as fallback -> Ideally we would never need this

Reply:
@thelostone-mc I think from a programming logic standpoint there is no case where this would be undefined after the if else block, so I think it should be removed

Reply:
removed ^_^